package.json

```diff
@@ -12,6 +12,7 @@
   "license": "ISC",
   "workspaces": [
     "packages/react",
-    "packages/react-dom"
+    "packages/react-dom",
+    "packages/vue"
```

Comment:
You can actually do `packages/*` instead of explicitly defining every package.

Reply:
When you run `npm init -w packages/vue` it automatically creates this entry in the root `package.json`. Thinking that if we define it as a glob then that's gonna get messed up a bit as the init command will still add a new entry for each new folder.

Reply:
Ack :+1: